### ISSUE TITLE
[1.16.3] Lack of registering configured feature

### ISSUE THREAD

**Issue by kwpugh** [Nic4Las/Minecraft-Enderite-Mod#18]
Hi,

I'm the author of Gobber mod:  https://github.com/kwpugh/Gobber

When I two mods are installed together, both of our ores in the end doo not spawn.   I'm not experienced enough to understand what might be causing it.

Can you give it a look to see if you can figure it out.

Regards.

**Comment by kwpugh** [comment]
I think there is something you need to do to register your configured feature.

![Uploading Screen Shot 2020-10-24 at 9.29.46 AM.png…]()

**Comment by TelepathicGrunt** [comment]
The issue is because Enderite is not registering their configuredfeatures which causes their unregistered cf to break the biome's codec and make it wipe out everyone else's registered cfs.

So the fix for this specific mod is to change this:
https://github.com/Nic4Las/Minecraft-Enderite-Mod/blob/c15c5606e4c7b86fd49598fc9c80561763d560de/src/main/java/net/enderitemc/enderitemod/init/WorldFeatures.java#L40-L48

to this:

```java
   ConfiguredFeature<?, ?> enderite_cf = Feature.ORE
                                                .withConfiguration(new OreFeatureConfig(fillerBlockType, state, 3))
                                                .withPlacement(Placement.field_242907_l.configure(countConfig))
                                                .func_242728_a().func_242731_b(count);

    Registry.register(WorldGenRegistries.field_243653_e, new ResourceLocation(Enderite.MODID, "enderite_cf"), enderite_cf);
    WorldGenRegistries.field_243657_i.forEach(biome -> {
                        if (biome.getCategory() == Biome.Category.THEEND) {
                                BiomeHandler handler = new BiomeHandler(biome);
                                handler.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, enderite_cf );
                        }
                });
```

**Comment by Nic4Las** [comment]
Thanks for the report and the fix. I'm currently unable to test this but i will let our Forge developer know of the fix.

**Comment by Pitti11** [comment]
Hopefully fixed in commit [b86863f](https://github.com/Nic4Las/Minecraft-Enderite-Mod/commit/b86863fe293cfebfadd4f14eb91adf9efdb79bef)
